lucet-runtime/include/lucet_types.h

```diff
@@ -16,6 +16,8 @@
 #include <ucontext.h>
 #endif
 
+#include <signal.h>
```

Comment:
This is only present because I was wildly flailing around trying to fix header issues that were caused by fiddling with an include path I shouldn't have. I left it because `lucet_types.h` references `siginfo_t` without including `signal.h`. This isn't an issue because `lucet.h` includes `signal.h` before `lucet_types.h`, but someone using `lucet_types.h` individually for some reason would be in for a surprise.

... I ought to make this a PR on its own.